modules/test/playwright/env/common.env.ts

```diff
@@ -0,0 +1,171 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {glob} from 'glob';
+import {copyFile, existsSync, mkdirSync, readFileSync} from 'node:fs'
+
+import {resolve} from 'path'
+
+import {executeBashScript} from './bashUtil.env';
+import {executeGradleTask} from './gradleUtil.env';
+import {getPropertiesFromFiles, getPropertyValue, writePropertiesFile} from './propertiesUtil.env';
+
+export function deployProjectClientExtensions() {
+	const clientExtensionsListFile = getPlaywrightProjectDir() + '/env/client-extensions.list';
```

Comment:
This is missing the base dir deployment (`playwright/env` dir). We need to be able to deploy things in both places because the login fixture (and in the future more fixtures and/or helpers) expect the base OSGi bundle to be deployed.

In general, everything should be multilevel: the client extensions, the OSGi bundles, portal.props, deploy, etc... 

In the case of portal-ext.properties, that's why I proposed to use `include-and-override` as I do in my PR so that we can deploy several files from several levels and tell DXP to merge them.

Reply:
Okay, I added this, but in a separate function.

I'll explain more in the new pull request since this one got really messy.